nlp-common/src/main/java/io/cdap/google/common/ClassifyContent.java

```diff
@@ -31,6 +34,23 @@ public ClassifyContent(String languageCode, EncodingType encoding, LanguageServi
     super(languageCode, encoding, language);
   }
 
+  @Override
+  protected StructuredRecord getRecordFromJson(String json) {
+    Schema schema = Schema.recordOf(ClassifyContent.class.getName(),
+                                    Schema.Field.of("categories", Schema.arrayOf(Schema.recordOf(
+                                      "categoriesRecord",
+                                      Schema.Field.of("name", Schema.of(Schema.Type.STRING)),
+                                      Schema.Field.of("confidence", Schema.of(Schema.Type.DOUBLE))
+                                    )))
+
+    );
+    JsonObject jsonObject = PARSER.parse(json).getAsJsonObject();
+
+    StructuredRecord.Builder builder = StructuredRecord.builder(schema);
```

Comment:
can you add comments here and in the other classes that show an example input with the corresponding output.